<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SR844Graphics</class>
 <widget class="QMainWindow" name="SR844Graphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBoxRefSource">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditResponse">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditError">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHarmonic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHarmonic">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Harmonic</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFrequency">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFrequency">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPhase">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPhase">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Phase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAutoPhase">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>AutoPhase</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxRefTriggerOutput">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAutoWideReverse">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>AutoWideRev</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxInputSignalZ">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxSensitivity">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAutoScale">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>AutoScale</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCloseReserveMode">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAutoReserve">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>AutoReserve</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTimeConstant">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFilterSlope">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSend">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSend">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRecieve">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutDataChannel1">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOutDataChannel1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxOutDataChannel1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutDataChannel2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOutDataChannel2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxOutDataChannel2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAutoZeroChannel1">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAutoZeroChannel2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutData">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOutData">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>170</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxOutData">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
